### PR TITLE
Get input calibration

### PR DESCRIPTION
**Pull request by AndersNordmark**

### CODE REVIEW

**Comment by AndersNordmark** [comment]
Bosch is requiring to be able to fetch calibration. I couldn't find a good way of doing that with the input API so I took some inspirations from the scene api.

File: kognic-io/kognic/io/resources/input/input.py [hunk 1]
```diff
@@ -88,3 +100,13 @@ def get_inputs_by_uuids(self, input_uuids: List[str]) -> List[Input]:
         body = dict(uuids=input_uuids)
         json_resp = self._client.post("v1/inputs/query", json=body)
         return [Input.from_json(js) for js in json_resp]
+
+    def get_input_calibration(self, input_uuid: str) -> Calibrations:
+        """
+        Gets inputs using input uuids
+
+        :param input_uuids: A UUID to filter inputs on
+        :return List: List of Inputs
```
Comment (simonbastas) [inline kognic-io/kognic/io/resources/input/input.py:109]: Fix comment

**Comment by olinoskar** [comment]
@AndersNordmark why not use this? 
```python
input = client.input.get_inputs_by_uuids(["the-uuid"])[0]
calibration = client.calibration.get_calibration(input.calibration_id)
```

**Comment by AndersNordmark** [comment]
Thanks Oskar! I missed that the calibration id was part of the input :) 
This works as a workaround
@simonbastas do you have any objections to closing this PR?

**Pull request closed without merging**